### DISCUSSION TITLE
Como adicionar meses com scroll na AppBar

### DISCUSSION THREAD

**Discussion by devgomes10** [Flutterando/forum#156] [Perguntas e Respostas]
Quero adicionar a opção de selecionar o mês através de setas localizado no app bar

![Screenshot_20230709-133646.png](https://github.com/Flutterando/forum/assets/115933691/daa1f80b-4131-43a6-9c90-78a5697e47a3)

**Comment by rodriguesJeff** [comment]
Uma possivel solucao eh dentro do title voce adicionar uma row e montar esse widget dentro, dai vc deixa o background da app bar transparente e o elevation como 0...
Pode ser meio gambi kk
Outra opcao seria nn ter app bar, apenas o body com um safearea e uma coluna, o primeiro item da coluna seria esse widget ai
